website/src/edit-word-form-context.tsx

```diff
@@ -63,6 +63,7 @@ export const FormProvider = (props: { children: ReactNode }) => {
             source: values.word["source"],
             commentary: values.word["commentary"],
             segments: updatedSegments,
+            translation: values.word["translation"],
           },
```

Comment:
Another confusing naming on our part. Should be `values.word["englishGloss"]` instead of `values.word["translation"]`.

You may run into a type mismatch here. I am looking into it but am blocked by another issue were having.

Reply:
I am getting a type error here.
```
Type 'readonly string[]' is not assignable to type 'InputMaybe<string>'.
```

Reply:
I can take a look at this since it seems like this will be annoying to debug without intellisense helping out.